python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -887,7 +887,8 @@ def _create_table_optimize_block(self, pserver_index, pserver_program,
         # create table optimize block in pserver program
         table_opt_op = [
             op for op in self.optimize_ops
-            if op.input("Param")[0] == self.table_name
+            if 'Param' in op.input_names and op.input("Param")[0] ==
```

Comment:
`self.optimize_ops` is all ops that have `optimize` role. Can you also add some unitest in `test_dist_transpiler.py` please, to check the dist lookup table part, Thanks.

Reply:
yes, I will add a unittest later. It must be in the unit test.